spec/system/provider_interface/provider_applications_sort_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec.feature 'Providers should be able to sort applications' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+
+  let(:tom_jones) { 'Tom Jones' }
+  let(:adam_jones) { 'Adam Jones' }
+  let(:jim_james) { 'Jim James' }
+  let(:bill_bones) { 'Bill Bones' }
```

Comment:
I think it would be a bit better to use string literals here as the spec depends on the string values themselves — i.e. if we redefined "bill bones" to "bill zones" the spec would still read correctly but it would fail

Reply:
👍